src/notfound/handler.js

```diff
@@ -11,11 +11,22 @@
  */
 
 import RUMAPIClient from '@adobe/spacecat-shared-rum-api-client';
-import { internalServerError, noContent } from '@adobe/spacecat-shared-http-utils';
+import { internalServerError, noContent, notFound } from '@adobe/spacecat-shared-http-utils';
+import { createUrl } from '@adobe/fetch';
+import { retrieveSiteByURL } from '../utils/data-access.js';
 import {
   getRUMUrl,
 } from '../support/utils.js';
 
+const AUDIT_TYPE = '404';
+// TODO export from spacecat-shared-rum-api-client
+const RUM_SOURCES = 'https://helix-pages.anywhere.run/helix-services/run-query@v3/rum-sources';
+export const RUM_DEFAULT_PARAMS = {
```

Comment:
rum-api-client already has `RUM_DEFAULT_PARAMS` and `RUM_SOURCES`; having two sources of truth is not good. Also this logic belongs to rum-api-client. Would you mind moving `create404Url` to rum-api-client?